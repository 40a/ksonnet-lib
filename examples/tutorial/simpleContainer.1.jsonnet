local kubeCore = import "../../kube/core.libsonnet";
local container = kubeCore.v1.container;

container.Default("nginx", "nginx:1.7.9") +
container.NamedPort("http", 80)