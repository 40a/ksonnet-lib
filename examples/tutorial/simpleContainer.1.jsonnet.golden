{
   "env": [ ],
   "image": "nginx:1.7.9",
   "imagePullPolicy": "Always",
   "name": "nginx",
   "ports": [
      {
         "containerPort": 80,
         "name": "http"
      }
   ],
   "volumeMounts": [ ]
}
